tests/test_feature_simple_cur.py

```diff
@@ -28,13 +28,13 @@ def test_restart(self):
         This test checks that the model can be restarted with a new instance
         """
 
-        ref_selector = CUR(n_features_to_select=self.X.shape[-1] - 2).fit(X=self.X)
+        ref_selector = CUR(n_features_to_select=self.X.shape[-1] - 3).fit(X=self.X)
         ref_idx = ref_selector.selected_idx_
 
         selector = CUR(n_features_to_select=1)
         selector.fit(self.X)
 
-        for i in range(self.X.shape[-1] - 2):
+        for i in range(self.X.shape[-1] - 3):
```

Comment:
Why does this need to be changed?

Reply:
Ah! This is because of the bounds of `scipy.sparse.linalg.svds`, which apparently are different than the bounds for `randomized_svds`. The test wasn't much different with the change, so I didn't look into it much.